apps/dav/lib/Connector/Sabre/File.php

```diff
@@ -306,7 +307,7 @@ public function put($data) {
 			}
 
 			// since we skipped the view we need to scan and emit the hooks ourselves
-			$storage->getUpdater()->update($internalPath);
+			$storage->getUpdater()->update($internalPath, null, ($writtenByteCount - $previousFileSize));
```

Comment:
To prevent undefined variable error ?

```suggestion
			$sizeDifference = null;
			if (isset($previousFileSize) && isset($previousFileSize)) {
				$sizeDifference = $writtenByteCount - $previousFileSize;
			}

			// since we skipped the view we need to scan and emit the hooks ourselves
			$storage->getUpdater()->update($internalPath, null, $sizeDifference);
```